### ISSUE TITLE
Add zsh plugins to common-utils feature

### ISSUE THREAD

**Issue by gund** [devcontainers/features#311]
Hi, I'm exploring dev containers and so far loving them!

I'm using `common-utils` feature as part of my dev container setup and wondering what is the proper way of adding a few ZSH plugins to it?

I've checked [the docs](https://github.com/devcontainers/features/tree/main/src/common-utils) of `common-utils` feature but it does not have an option to specify list of plugins.

Any ideas?

**Comment by bamurtaugh** [comment]
Hi @gund, thanks for opening, and we're so happy to hear you're loving dev containers!

There's currently discussion over potentially splitting up the common feature: https://github.com/devcontainers/features/issues/67. So at this time, there aren't plans to add more to it.

You could certainly add zsh to your dev container through a separate Feature or install step. Depending on your setup, if you're using a specific editor like VS Code, there's a `vscode.extensions` property in devcontainer.json where you can specify extensions to install.

**Comment by gund** [comment]
Hey @bamurtaugh thanks for letting me know about the plans for the common feature.

Actually I already created a [custom feature](https://github.com/gund/ng-dynamic-component/blob/master/.devcontainer/zsh-plugins/devcontainer-feature.json) that allows to [configure (oh-my-)zsh plugins](https://github.com/gund/ng-dynamic-component/blob/d25fc9b780fcecd11180d28bec709825a4db6e71/.devcontainer/devcontainer.json#L16-L19) from a `devcointainer.json`, so maybe I could contribute it to the zsh feature that you are going to split from the common feature?

Regarding `vscode.extensions` property I'm not quite sure how this would help as those are VSCode extensions but I need a way to extend actual devcontainer image to install more zsh plugins, or am I missing something?

**Comment by gund** [comment]
Thanks for the guidance!

I figured that creating a brand new repo would be probably too much for my feature so I actually created a [PR to devcontainer-contrib repo](https://github.com/devcontainers-contrib/features/pull/198) and since it's already added to the collection you mentioned above, my feature should be automatically discoverable I guess 😊

I think we can resolve this issue.

Thanks for your help!
